pkgs/applications/networking/cluster/code-generator/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub, git }:
+
+buildGoModule rec {
+  pname = "code-generator";
+  version = "0.25.4";
+
+  src = fetchFromGitHub {
+    owner = "kubernetes";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-GKF6DXvyZujInOZbV0ePUu71BEl1s/chNTN1PucdIYw=";
+  };
+
+  vendorHash = "sha256-zjgTtGen6a8TPi/DrwheTS1VQ+hd+KI7UHoyMZ4W4+k=";
+
```

Comment:
```suggestion

  ldflags = [ "-s" "-w" ];

```

Reply:
I have only added `ldflags` when I want to do a `-X path/to/pkg.symbol=value`, is this required by default now? (and if so, why is it not the default when you leave `ldflags = null`)

Reply:
See #177698.